examples/web/package.json

```diff
@@ -27,28 +27,25 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "babel-loader": "^8.2.2",
+    "@babel/core": "^7.21.8",
+    "babel-loader": "^8.3.0",
     "ts-loader": "^6.2.2",
-    "webpack": "^4.44.2",
+    "webpack": "^4.46.0",
     "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0",
+    "webpack-dev-server": "^3.11.3",
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.0.3",
-    "@opentelemetry/auto-instrumentations-web": "~0.26.0",
-    "@opentelemetry/context-zone": "~1.0.0",
-    "@opentelemetry/core": "~1.0.0",
-    "@opentelemetry/exporter-otlp-http": "~0.26.0",
+    "@opentelemetry/api": "^1.4.1",
+    "@opentelemetry/auto-instrumentations-web": "^0.32.2",
+    "@opentelemetry/context-zone": "^1.13.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.39.1",
     "@opentelemetry/instrumentation": "~0.26.0",
```

Comment:
why update some deps but not others? This likely results in duplicates in node_modules folder

Reply:
I just missed some. Will update.

Reply:
Got them all now.